src/components/Table.tsx

```diff
@@ -67,6 +57,14 @@ const BaseTable = styled.table`
 			&:nth-of-type(even) {
 				background-color: #f8f8f8;
 			}
+
+			&[data-checked='true'] {
+				${highlightStyle};
+			}
+
+			&:hover {
+				${(props: any) => (!!props.onRowClick ? highlightStyle : '')};
```

Comment:
Loved this :+1: 
My only request in this review: Would it make sense to not add the `theme.colors.info.main` part in front for the hover effect and use it only for selection?

Reply:
Yep I think that makes sense